internal/pkg/skuba/addons/dex.go

```diff
@@ -164,6 +164,7 @@ metadata:
   labels:
     app: oidc-dex
 spec:
+  revisionHistoryLimit: 3
```

Comment:
Already in https://github.com/SUSE/skuba/blob/c8998e8/internal/pkg/skuba/addons/dex.go#L181